tsconfig.json

```diff
@@ -3,14 +3,23 @@
     "strict": true,
     "allowSyntheticDefaultImports": true,
     "jsx": "react-native",
-    "lib": ["dom", "esnext"],
+    "lib": [
```

Comment:
Just wondering, what triggered these?

Reply:
tested, both passes somehow :| , reverted to the previous version